trie/patriciaMerkleTrie.go

```diff
@@ -481,7 +481,7 @@ func (tr *patriciaMerkleTrie) GetAllLeavesOnChannel(
 			ctx,
 		)
 		if err != nil {
-			writeInChanNonBlocking(leavesChannels.ErrChan, err)
+			leavesChannels.ErrChan.WriteInChanNonBlocking(err)
```

Comment:
the old `writeInChanNonBlocking` method is now used only in tests. Move the implementation directly in the export_test.go file (in WriteInChanNonBlocking function)?

Reply:
Done.